app/uk/gov/hmrc/plasticpackagingtax/returns/models/financials/PPTFinancials.scala

```diff
@@ -50,3 +58,15 @@ final case class PPTFinancials(
 object PPTFinancials {
   implicit val format: OFormat[PPTFinancials] = Json.format[PPTFinancials]
 }
+
+object CurrencyFormatters {
+
+  def formatCurrencyAmount(amount: BigDecimal): String = {
```

Comment:
Maybe something explicit about the intended formating like formatAsPoundsAndPennies but not blocking.